cypress/support/pages/articleEditor.pageObject.js

```diff
@@ -0,0 +1,28 @@
+import PageObject from '../PageObject';
+
+class ArticleEditorPageObject extends PageObject {
+  url = '/#/editor';
+
+  typeTitle(title) {
+    cy.getByDataCy('articleTitle')
+      .clear()
+      .type(title);
+  }
+
+  typeDescription(desc) {
+    cy.getByDataCy('articleDescription')
+      .type(desc);
```

Comment:
Before typing a new description, the existing text in the description field should be cleared to prevent appending to the existing text.